manage

```diff
@@ -499,7 +499,7 @@ if ! isInstalled ${SED_EXE}; then
 
   cat <<-EOF
   The recommended approach to installing the required package(s) is to use either [Homebrew](https://brew.sh/) (MAC)
-  or [Chocolatey](https://chocolatey.org/) (Windows).  For more information visit https://mikefarah.github.io/yq/
+  or [Chocolatey](https://chocolatey.org/) (Windows) or [GitHub](https://github.com/mikefarah/yq/) (Linux).  For more information visit https://mikefarah.github.io/yq/
```

Comment:
These should be the instructions for installing `sed`.  The existing reference to `yq` here was a copy and paste mistake.